pkg/util/util.go

```diff
@@ -102,8 +102,11 @@ func NeedVirtioNetDevice(vmi *v1.VirtualMachineInstance, allowEmulation bool) bo
 
 func NeedTunDevice(vmi *v1.VirtualMachineInstance) bool {
 	return (len(vmi.Spec.Domain.Devices.Interfaces) > 0) ||
-		(vmi.Spec.Domain.Devices.AutoattachPodInterface == nil) ||
-		(*vmi.Spec.Domain.Devices.AutoattachPodInterface == true)
+		IsPodNetworkAutoattaced(vmi.Spec.Domain.Devices.AutoattachPodInterface)
+}
+
+func IsPodNetworkAutoattaced(autoattachPodInterface *bool) bool {
```

Comment:
It is unfortunate, but I do not think we should add to any files or packages that are named `util` or `utils` (or `common` and anything alike).

Reply:
We can refactor but not on this PR please

Reply:
Personally I think that code reuse, and a follow PR that will refactor it (and have already the new function),
is better than not adding a function to util, it also allows people to use the function easily instead of thinking about the conditions needed, but will inline it